### ISSUE TITLE
Select2 4.0

### ISSUE THREAD

**Issue by kevin-brown** [iStefo/ember-select-2#51]
We're in the process of finalizing the 4.0 release and are aiming to have it done by the end of this month at the latest.

https://select2.github.io/

Since I'm not very familiar with this plugin, I can't really judge how easy it is going to be to migrate over to the new version.

**Comment by iStefo** [comment]
Thanks for letting me know! We probably won't make the jump to 4.0 until the next release. because I have very little time maintaining this project. But this will be on the roadmap for sure.

**Comment by drewcovi** [comment]
Any ideas on when we can see this? Select2 4.0 looks pretty stellar.

**Comment by kevin-brown** [comment]
Supposedly this works as-is in AngularJS because Select2 4.0 interacts directly with the `<select>` element instead of requiring hacks with the `<input />` element.

Has anyone tried it out without using `ember-select-2` to see if this is the case for Ember as well?

**Comment by fantasticfears** [comment]
@kevin-brown I am experimenting select2 right now. Since select2 4.0 interacts with `<select>`, I am going to extend the `Ember.Select`. Releasing the code soon, hope it can be finally merged into master.

**Comment by albertolobrano** [comment]
@fantasticfears  did you manage to run it ?  i am getting this error all the time : Uncaught Error: Could not find module ./compat/inputData

**Comment by fantasticfears** [comment]
@albertolobrano Yes, the integration is fairly easy. But I dropped many custom binding built by @iStefo. I don't know whether it's a good thing.

I think @iStefo should decide what to do with the custom bindings. Since when following `Ember.Select`, many things have to change.

**Comment by basz** [comment]
Isn't Ember.Select suppost to be deprecated?

**Comment by fantasticfears** [comment]
@basz It's deprecated since 1.13. This can be helpful: http://emberjs.com/deprecations/v1.x/#toc_ember-select

**Comment by phammer** [comment]
@iStefo : What's the plan with the upgrade or even more general with this project? Would be great to still have a simple select plugin for ember cli that is based on a solid ground like select2.
